core/domain/platform_feature_services.py

```diff
@@ -245,7 +245,7 @@ def _create_evaluation_context_for_server() -> (
     )
 
 
-def _evaluate_feature_flag_values_for_context(
+def _evaluate_feature_flag_configs_for_context(
```

Comment:
Isn't the stuff here supposed to be evaluating the values and not the configs? The configs are what is stored in the datastore.

Reply:
Updated